packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/util.ts

```diff
@@ -144,11 +151,59 @@ export async function generateSpokenMathDetails(mathString: string) {
     // to look for individual math blocks.
     if (firstSection.type === "paragraph") {
         for (const piece of firstSection.content) {
-            piece.type === "math"
-                ? (convertedSpeech += engine.texToSpeech(piece.content))
-                : (convertedSpeech += piece.content);
+            switch (piece.type) {
+                case "math":
+                    convertedSpeech += engine.texToSpeech(piece.content);
+                    break;
+                case "unescapedDollar":
+                    // If the unescaped dollar had a closing pair to define
+                    // math, it would have been caught by the "math" case above.
+                    // Since this unescaped dollar is caught here, we can
+                    // assume it is alone and used as as a literal dollar sign.
+                    convertedSpeech += "$";
```

Comment:
This change is from another PR, right? You might need to change the base branch for this PR in GitHub (or land the other PR and then rebase these changes on `main`).

Reply:
There are two util functions that have a similar branch! The are stuff is in this file. The visual TeX stuff is in the other `utils` file.